### ISSUE TITLE
Improve logging when node joins itself

### ISSUE THREAD

**Issue by renatocaval** [akka/akka#25279]
When a node joins itself, the logging message is not so obvious. 

There are two (possibly more) indications that a node is join itself. 

In DEBUG level:
```
[DEBUG] Couldn't join other seed nodes, will join myself. seed-nodes=[...]
```

In INFO level:
```
[INFO] Cluster Node [akka.tcp://cluster@127.0.0.1:2551] - 
       Node [akka.tcp://cluster@127.0.0.1:2551] is JOINING, roles [dc-default]
```

While when the node join an existing cluster, we get a pretty clear message in INFO level.

```
[INFO] Cluster Node [akka.tcp://cluster@127.0.0.1:2551] - 
       Welcome from [akka.tcp://cluster@127.0.0.1:2552]
```

I think that when a node joins itself it's a pretty important event. Either we are starting a new fresh cluster or we are creating an island.

We should instead either log the first message in INFO level or make the second one more explicit. One needs to interpret and have knowledge about the internals to understand that when you see `Cluster Node [node1] - Node [node1] is JOINING` means that the node is joining itself. 

Alternatively, we could change the message to  
```
Cluster Node [node1] - Node [node1] is JOINING itself
or 
Cluster Node [node1] - Node [node1] is JOINING itself and forming a new cluster
``` 

That would make it more explicit that the node is starting a new cluster of its own.

**Comment by johanandren** [comment]
Thinking some about this I agree, and I like the last version where it explicitly says that it is forming a new cluster (useful for users who have *no* idea at all what is going on)

**Comment by patriknw** [comment]
I agree, would be good with a unique log message to search for in logs. Would you be able to create PR, @renatocaval ?

**Comment by Seetaramayya** [comment]
I would like to pick. I'll try this

**Comment by chbatey** [comment]
Thanks @Seetaramayya
